test/avocado/run-tests

```diff
@@ -134,7 +134,7 @@ def run_avocado_tests(machine, avocado_tests, print_failed=True, env=[]):
 
 def copy_avocado_logs(machine, title):
     if machine and machine.address:
-        dir = "%s-%s.avocado" % (machine.address, title)
+        dir = os.path.join(testinfra.TEST_DIR, "tmp", "%s-%s.avocado" % (machine.address, title))
```

Comment:
Is it worth noting here that we don't use `tempfile.mkdtemp()` because we want to have the nice name in the archive?

Also, what if we just tar this on the remote / virtual machine and only download the resulting file?